atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraVerifier.java

```diff
@@ -189,6 +189,11 @@ private static boolean keyspaceAlreadyExists(InetSocketAddress host, CassandraKe
         try {
             Client client = CassandraClientFactory.getClientInternal(host, config);
             client.describe_keyspace(config.getKeyspaceOrThrow());
+            CassandraKeyValueServices.waitForSchemaVersions(
+                    config,
+                    client,
+                    "(adding the initial empty keyspace)",
```

Comment:
The message should be updated here - we didn't add the keyspace, we're checking for its existence.